### ISSUE TITLE
Error when creating first account

### ISSUE THREAD

**Issue by ylecuyer** [Attendize/Attendize#71]
After installing, I have the following error when I attemp to create the first account :

```
[2016-04-18 22:38:35] production.ERROR: exception 'PDOException' with message 'SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "name" violates not-null constraint
DETAIL:  Failing row contains (1, Yoann, Lecuyer, yoann.lecuyer@yopmail.com, 30, null, null, 2, 2016-04-18 22:38:35, 2016-04-18 22:38:35, null, null, null, null, null, null, null, null, null, null, null, f, f, f, null, null, null, null, null, 1).' in /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php:335
Stack trace:
#0 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(335): PDOStatement->execute(Array)
#1 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(666): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\PostgresConnection), 'insert into "ac...', Array)
#2 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(629): Illuminate\Database\Connection->runQueryCallback('insert into "ac...', Array, Object(Closure))
#3 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(342): Illuminate\Database\Connection->run('insert into "ac...', Array, Object(Closure))
#4 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(312): Illuminate\Database\Connection->select('insert into "ac...', Array, false)
#5 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/PostgresProcessor.php(20): Illuminate\Database\Connection->selectFromWriteConnection('insert into "ac...', Array)
#6 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1963): Illuminate\Database\Query\Processors\PostgresProcessor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into "ac...', Array, 'id')
#7 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#8 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1337): call_user_func_array(Array, Array)
#9 /home/deploy/attendize/bootstrap/cache/compiled.php(10791): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#10 /home/deploy/attendize/bootstrap/cache/compiled.php(10791): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#11 /home/deploy/attendize/bootstrap/cache/compiled.php(10780): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#12 /home/deploy/attendize/bootstrap/cache/compiled.php(10731): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder), Array)
#13 /home/deploy/attendize/bootstrap/cache/compiled.php(10391): Illuminate\Database\Eloquent\Model->save()
#14 /home/deploy/attendize/app/Http/Controllers/UserSignupController.php(53): Illuminate\Database\Eloquent\Model::create(Array)
#15 [internal function]: App\Http\Controllers\UserSignupController->postSignup(Object(Illuminate\Http\Request))
#16 /home/deploy/attendize/bootstrap/cache/compiled.php(9363): call_user_func_array(Array, Array)
#17 /home/deploy/attendize/bootstrap/cache/compiled.php(9425): Illuminate\Routing\Controller->callAction('postSignup', Array)
#18 /home/deploy/attendize/bootstrap/cache/compiled.php(9405): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\UserSignupController), Object(Illuminate\Routing\Route), 'postSignup')
#19 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 /home/deploy/attendize/app/Http/Middleware/RedirectIfAuthenticated.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: App\Http\Middleware\RedirectIfAuthenticated->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#24 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 /home/deploy/attendize/bootstrap/cache/compiled.php(9882): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 /home/deploy/attendize/bootstrap/cache/compiled.php(9406): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 /home/deploy/attendize/bootstrap/cache/compiled.php(9393): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\UserSignupController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'postSignup')
#30 /home/deploy/attendize/bootstrap/cache/compiled.php(8474): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\Http\Contro...', 'postSignup')
#31 /home/deploy/attendize/bootstrap/cache/compiled.php(8461): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#32 /home/deploy/attendize/bootstrap/cache/compiled.php(8175): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#33 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#34 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#35 /home/deploy/attendize/app/Http/Middleware/CheckInstalled.php(26): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#36 [internal function]: app\Http\Middleware\CheckInstalled->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#38 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#40 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#41 /home/deploy/attendize/bootstrap/cache/compiled.php(9882): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#42 /home/deploy/attendize/bootstrap/cache/compiled.php(8176): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#43 /home/deploy/attendize/bootstrap/cache/compiled.php(8167): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#44 /home/deploy/attendize/bootstrap/cache/compiled.php(8157): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#45 /home/deploy/attendize/bootstrap/cache/compiled.php(2400): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#46 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#47 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#48 /home/deploy/attendize/app/Http/Middleware/GeneralChecks.php(25): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#49 [internal function]: app\Http\Middleware\GeneralChecks->handle(Object(Illuminate\Http\Request), Object(Closure))
#50 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#51 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#52 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#53 /home/deploy/attendize/bootstrap/cache/compiled.php(3176): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#54 /home/deploy/attendize/app/Http/Middleware/VerifyCsrfToken.php(24): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#55 [internal function]: App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#56 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#57 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#58 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#59 /home/deploy/attendize/bootstrap/cache/compiled.php(13328): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#60 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#61 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#62 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#63 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#64 /home/deploy/attendize/bootstrap/cache/compiled.php(11876): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#65 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#66 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#67 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#68 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#69 /home/deploy/attendize/bootstrap/cache/compiled.php(13063): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#70 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#71 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#72 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#73 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#74 /home/deploy/attendize/bootstrap/cache/compiled.php(13000): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#75 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#76 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#77 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#78 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#79 /home/deploy/attendize/bootstrap/cache/compiled.php(3237): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#80 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#81 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#82 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#83 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#84 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#85 /home/deploy/attendize/bootstrap/cache/compiled.php(9882): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#86 /home/deploy/attendize/bootstrap/cache/compiled.php(2347): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#87 /home/deploy/attendize/bootstrap/cache/compiled.php(2331): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#88 /home/deploy/attendize/public/index.php(52): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#89 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "name" violates not-null constraint
DETAIL:  Failing row contains (1, Yoann, Lecuyer, yoann.lecuyer@yopmail.com, 30, null, null, 2, 2016-04-18 22:38:35, 2016-04-18 22:38:35, null, null, null, null, null, null, null, null, null, null, null, f, f, f, null, null, null, null, null, 1). (SQL: insert into "accounts" ("email", "first_name", "last_name", "currency_id", "timezone_id", "updated_at", "created_at") values (yoann.lecuyer@yopmail.com, Yoann, Lecuyer, 2, 30, 2016-04-18 22:38:35, 2016-04-18 22:38:35) returning "id")' in /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php:673
Stack trace:
#0 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(629): Illuminate\Database\Connection->runQueryCallback('insert into "ac...', Array, Object(Closure))
#1 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(342): Illuminate\Database\Connection->run('insert into "ac...', Array, Object(Closure))
#2 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Connection.php(312): Illuminate\Database\Connection->select('insert into "ac...', Array, false)
#3 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Query/Processors/PostgresProcessor.php(20): Illuminate\Database\Connection->selectFromWriteConnection('insert into "ac...', Array)
#4 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1963): Illuminate\Database\Query\Processors\PostgresProcessor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into "ac...', Array, 'id')
#5 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#6 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1337): call_user_func_array(Array, Array)
#7 /home/deploy/attendize/bootstrap/cache/compiled.php(10791): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#8 /home/deploy/attendize/bootstrap/cache/compiled.php(10791): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#9 /home/deploy/attendize/bootstrap/cache/compiled.php(10780): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#10 /home/deploy/attendize/bootstrap/cache/compiled.php(10731): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder), Array)
#11 /home/deploy/attendize/bootstrap/cache/compiled.php(10391): Illuminate\Database\Eloquent\Model->save()
#12 /home/deploy/attendize/app/Http/Controllers/UserSignupController.php(53): Illuminate\Database\Eloquent\Model::create(Array)
#13 [internal function]: App\Http\Controllers\UserSignupController->postSignup(Object(Illuminate\Http\Request))
#14 /home/deploy/attendize/bootstrap/cache/compiled.php(9363): call_user_func_array(Array, Array)
#15 /home/deploy/attendize/bootstrap/cache/compiled.php(9425): Illuminate\Routing\Controller->callAction('postSignup', Array)
#16 /home/deploy/attendize/bootstrap/cache/compiled.php(9405): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\UserSignupController), Object(Illuminate\Routing\Route), 'postSignup')
#17 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#18 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 /home/deploy/attendize/app/Http/Middleware/RedirectIfAuthenticated.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: App\Http\Middleware\RedirectIfAuthenticated->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 /home/deploy/attendize/bootstrap/cache/compiled.php(9882): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 /home/deploy/attendize/bootstrap/cache/compiled.php(9406): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 /home/deploy/attendize/bootstrap/cache/compiled.php(9393): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\UserSignupController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'postSignup')
#28 /home/deploy/attendize/bootstrap/cache/compiled.php(8474): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\Http\Contro...', 'postSignup')
#29 /home/deploy/attendize/bootstrap/cache/compiled.php(8461): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#30 /home/deploy/attendize/bootstrap/cache/compiled.php(8175): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#32 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 /home/deploy/attendize/app/Http/Middleware/CheckInstalled.php(26): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#34 [internal function]: app\Http\Middleware\CheckInstalled->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#36 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#39 /home/deploy/attendize/bootstrap/cache/compiled.php(9882): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#40 /home/deploy/attendize/bootstrap/cache/compiled.php(8176): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#41 /home/deploy/attendize/bootstrap/cache/compiled.php(8167): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#42 /home/deploy/attendize/bootstrap/cache/compiled.php(8157): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#43 /home/deploy/attendize/bootstrap/cache/compiled.php(2400): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#45 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#46 /home/deploy/attendize/app/Http/Middleware/GeneralChecks.php(25): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#47 [internal function]: app\Http\Middleware\GeneralChecks->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#49 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#51 /home/deploy/attendize/bootstrap/cache/compiled.php(3176): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#52 /home/deploy/attendize/app/Http/Middleware/VerifyCsrfToken.php(24): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#53 [internal function]: App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#54 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#55 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#56 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#57 /home/deploy/attendize/bootstrap/cache/compiled.php(13328): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#58 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#59 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#60 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#61 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#62 /home/deploy/attendize/bootstrap/cache/compiled.php(11876): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#63 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#64 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#65 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#66 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#67 /home/deploy/attendize/bootstrap/cache/compiled.php(13063): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#68 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#69 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#70 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#71 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#72 /home/deploy/attendize/bootstrap/cache/compiled.php(13000): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#73 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#74 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#75 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#76 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#77 /home/deploy/attendize/bootstrap/cache/compiled.php(3237): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#78 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#79 /home/deploy/attendize/bootstrap/cache/compiled.php(9892): call_user_func_array(Array, Array)
#80 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#81 /home/deploy/attendize/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#82 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#83 /home/deploy/attendize/bootstrap/cache/compiled.php(9882): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#84 /home/deploy/attendize/bootstrap/cache/compiled.php(2347): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#85 /home/deploy/attendize/bootstrap/cache/compiled.php(2331): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#86 /home/deploy/attendize/public/index.php(52): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#87 {main}  
```

**Comment by ylecuyer** [comment]
All not null columns of the accounts table should be set to nullable and the phone column of users too.

**Comment by ylecuyer** [comment]
Fixed in PR #77
